### ISSUE TITLE
[MVT] WEBGL -GPU Support improvement

### ISSUE THREAD

**Issue by davidoesch** [geoadmin/mf-geoadmin3#4859]
Current ol6 implementation does not use GPU as well as mapbox for WebGL functions
This results in a battery drain heating up of the mobile devices 

pls check with ol6 dev community regarding improvement of GPU usage

**Comment by boecklic** [comment]
**tl;dr**: native webgl support in openlayers is out of scope for mid-term future. Currently the only way of bringing webgl power to OL is by using mapbox-gl as rendering engine for a OL-layer

Quote/summary from telco with c2c (refer to https://ltboc.infra.bgdi.ch/notebooks/mvt/roadmap_openlayers.html#Roadmap): 
"The WebGL code in OL was removed because it was not well tested and not really maintained by the community. Competence in WebGL at c2c the OL community is currently very limited. Integrating WebGL code into a framework is fairly complex and puts high demands on knowhow and money (Having a WebGL renderer developed properly would require at least ~ 3Mio. CHF, rough estimation by c2c). The experimental WebGL code in OL caused a lot of work in the past with limited outcome, it's simply too much work. Therefore, canvas-only rendering and still is more developed and maintained and has seen major improvements over the last six months. Nevertheless, certain refactoring was done to simplify the integration of renderers using WebGL, mainly by offloading the rendering duty to the layer (such each layer can have a different renderer). The heatmap renderer is an example of using WebGL to render simple primitives (points), but is far away from being capable of rendering complex geometries like polygons, fonts etc. typically found in vectortiles, it's just "a black hole". But the opinion in the community about a WebGL renderer in OL is not uniform, some like AHocevar are in favour of a pure canvas-only OL framework, others like TSchaub are much more in favour of WebGL."

**Comment by davidoesch** [comment]
Imho : this was part of the research contract

**Comment by boecklic** [comment]
@davidoesch imho this can be closed. GPU is now used via mapbox-gl, native GPU support for MVT in ol6 is future
